app/models/nonbillable_account.rb

```diff
@@ -10,7 +10,7 @@ def self.singleton_instance
     first || create
   end
 
-  def account_open?(_account_number)
+  def account_open?(_account_number, fulfillment_time: nil)
```

Comment:
Sorry, forgot to add a nit comment: can we prefix with `_` as this param isn't used?

Reply:
I agree with Leti here

Reply:
@giladshanan @LeticiaErrandonea since `fulfillment_time` is a named parameter, putting `_` in front gives an error (`ArgumentError: unknown keyword: :fulfillment_time`), so I updated the method to accept anything.